@here/harp-map-controls/lib/CameraKeyTrackAnimation.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { MapView, MapViewEventNames, MapViewUtils } from "@here/harp-mapview";
+import { LoggerManager } from "@here/harp-utils";
+import THREE = require("three");
+
+import { CameraAnimation } from "./MapAnimations";
+import { CameraPose, CameraPoseOptions } from "./Utils";
+
+const logger = LoggerManager.instance.create("CameraKeyTrackAnimation");
+
+export interface ControlPointOptions extends CameraPoseOptions {
+    timestamp: number;
+}
+/**
+ * A ControlPoint used to defined the Track of the [[CameraKeyTrackAnimation]]
+ */
+export class ControlPoint extends CameraPose {
+    timestamp: number = 0;
+    constructor(options: ControlPointOptions) {
+        super(options);
+        this.timestamp = options.timestamp;
+    }
+}
+
+export interface CameraKeyTrackAnimationOptions {
+    controlPoints: ControlPoint[];
+    repeat?: boolean;
```

Comment:
Clarify `repeat`, `loop` and `repetitions` ...

Reply:
done